src/cli.zig

```diff
@@ -175,7 +194,15 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
         },
         .format => {
             if (addresses != null) fatal("--addresses: supported only by 'start' command", .{});
-            if (memory != null) fatal("--memory: supported only by 'start' command", .{});
+            if (cache_accounts != null) {
+                fatal("--cache-accounts: supported only by 'start' command", .{});
+            }
+            if (cache_transfers != null) {
+                fatal("--cache-transfers: supported only by 'start' command", .{});
+            }
+            if (cache_transfers_posted != null) {
+                fatal("--cache-transfers-posted: supported only by 'start' command", .{});
+            }
```

Comment:
similarly, this is already quite repetetive, and we'll probably grow a number of options over time